core/src/main/resources/META-INF/native-image/com.linecorp.armeria/armeria/predefined-classes-config.json

```diff
@@ -1,7 +1,4 @@
-[
-  {
-    "type":"agent-extracted",
-    "classes":[
```

Comment:
Do we need a break line here?

Reply:
These files are automatically generated, so I think it doesn't really matter as long as it's auto-formatted. This PR actually added an auto-formatting step, which is why we see a lot of diffs in this file.